### DISCUSSION TITLE
Python tool : Create an update query based on enumerations.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#322] [Show and tell]
the tool uses a PyQT interface to create the "update" custom query, proposing the enumerations contained in the database.
- Copy/Paste interaction with BiblioteQ

- Takes into account the variable: BIBLIOTEQ_DATABASE_NAME for the sqlite database file

- Launch by argument --special-executable $HOME/NOMBIB/11_lancement_massupdate.sh

- For the moment in French only.

[11_BQMassUpdate-version_2023-08-10.tar.gz](https://github.com/textbrowser/biblioteq/files/12535647/11_BQMassUpdate-version_2023-08-10.tar.gz)

**Comment by FriedrichFroebel** [comment]
Just a general thought: If you create a repository in your GitHub account and upload the source code there without doing ZIP uploads like in this discussion, it would be much easier for others to have a look at it and watch for future updates (or even contribute a non-French version etc. if someone is interested in this).

**Reply by meteos77** [reply]
Hello and thank you for your constructive advice. Unfortunately I'm not a developer and git and github are quite complicated for me. Lack of time and desire too.

I'd rather make .tar.gz available than keep it for others to use if they have the same problem as me. 
(ditto for my pan2bq.py tool, which transforms an unimarc file (.pan) into a .csv for import into BQ).

**Reply by meteos77** [reply]
test on another site because github is not in my language:
https://codeberg.org/meteos77/BQMassUpdate

**Comment by textbrowser** [comment]
Discussions do not offer histories. :) They cannot be forked, reviewed, compared, reverted, logged, analyzed. They are fixed and only exist within GitHub. :)
